help/sources/tutorials/ui/create/crm/sap-hybris-subscription-billing-customers-and-contacts.md

```diff
@@ -0,0 +1,249 @@
+
+### Existing account {#existing-account}
+
+To use an existing account, select the [!DNL SAP Hybris] account you want to create a new dataflow with, then select **[!UICONTROL Next]** to proceed.
+
+![Platform UI screenshot to connect SAP Hybris account with an existing account](../../../../images/tutorials/create/crm/sap-hybris-subscription-billing-customers-and-contacts/existing.png)
+
+### New account {#new-account}
+
+If you are creating a new account, select **[!UICONTROL New account]**, and then provide a name, an optional description, and your credentials. When finished, select **[!UICONTROL Connect to source]** and then allow some time for the new connection to establish.
+
+![Platform UI screenshot to connect SAP Hybris account with a new account](../../../../images/tutorials/create/crm/sap-hybris-subscription-billing-customers-and-contacts/new.png)
+
+### Select data {#select-data}
+
+Finally, you must select the object type that you want to ingest to Platform.
+
+| Object type | Description |
+| --- | --- |
+| `Customers` | The entities who have subscriptions. |
+| `Contacts` | The contact details for customers. |
+
+>[!BEGINTABS]
+
+>[!TAB Customers]
+
+![Platform UI screenshot for SAP Hybris showing configuration with Customers option selected](../../../../images/tutorials/create/crm/sap-hybris-subscription-billing-customers-and-contacts/configuration-customers.png)
+
+>[!TAB Contacts]
+
+![Platform UI screenshot for SAP Hybris showing configuration with Contacts option selected](../../../../images/tutorials/create/crm/sap-hybris-subscription-billing-customers-and-contacts/configuration-contacts.png)
+
+>[!ENDTABS]
+
+## Next steps {#next-steps}
+
+By following this tutorial, you have established a connection to your [!DNL SAP Hybris] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](../../dataflow/crm.md).
+
+## Additional resources {#additional-resources}
+
+The sections below provide additional resources that you can refer to when using the [!DNL SAP Hybris] source.
+
+### Mapping {#mapping}
+
+The [!UICONTROL Mapping] step appears, providing you with an interface to map the source fields from your source schema to their appropriate target XDM fields in the target schema.
+
+Platform provides intelligent recommendations for auto-mapped fields based on the target schema or dataset that you selected. You can manually adjust mapping rules to suit your use cases. Based on your needs, you can choose to map fields directly, or use data prep functions to transform source data to derive computed or calculated values. For comprehensive steps on using the mapper interface and calculated fields, see the [Data Prep UI guide](../../../../../data-prep/ui/mapping.md).
+
+Based on the `object_type` selection *(either `customers` or `contacts`)* and your schema the mappings would be as below. Setup the desired mappings before proceeding to the next stage.
```

Comment:
need a comma after "schema" and "Setup" in the second sentence should be "Set up"